src/containerapp/azext_containerapp/custom.py

```diff
@@ -2629,3 +2630,70 @@ def list_environment_telemetry_otlp(cmd,
 
     return containerapp_env_def
 
+
+def list_replica_containerappsjob(cmd, resource_group_name, name, execution=None):
+    if execution is None:
+        executions = ContainerAppsJobPreviewClient.get_executions(cmd=cmd, resource_group_name=resource_group_name, name=name)
+        execution = executions['value'][0]['name']
+        logger.warning('No execution specified. Using the latest execution: %s', execution)
```

Comment:
As the execution is not required for command, should this log be `info` level, rather than `warning`?

Reply:
Though not required, we would like the user to specify execution when possible.

Reply:
Why do we want to pick random/first execution in the list vs having execution name to be mandatory?

More user friendly way, would be to list last 10 executions and ask customer to choose from it